portfolio/src/main/webapp/script.js

```diff
@@ -25,3 +25,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//add Hello Nzinga! to the page
+async function getHelloNzingaUsingAsyncAwait() {
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace): Please add a blank line between the methods

```suggestion

// Add Hello Nzinga! to the page
async function getHelloNzingaUsingAsyncAwait() {
```